### PR TITLE
Fix aws_s3_ bucket_policy #to_s method to reflect object

### PR DESCRIPTION
**Pull request by vsuzdaltsev**
### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec AWS](https://github.com/inspec/inspec-aws/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [ ] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [ ] All Unit Tests pass
- [ ] `rake lint` passes
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by vsuzdaltsev** [comment]
This way seems like a bit more readable image

then 
image

**Comment by vsuzdaltsev** [comment]
Also I would be happy to understand why only the first element of a policy statement is parsed here
image

File: libraries/aws_s3_bucket_policy.rb [hunk 1]
```diff
@@ -33,6 +33,6 @@ def resource_id
   end
 
   def to_s
-    "S3 Bucket Name: #{@display_name}"
+    "S3 Bucket Policy #{@display_name}"
```
Comment (soumyo13) [inline libraries/aws_s3_bucket_policy.rb:36]: We can't change 'Name' to 'Policy' as @display_name returns the name of the bucket.

**Review by soumyo13 — CHANGES_REQUESTED** [review]

**Comment by soumyo13** [comment]
> Also I would be happy to understand why only the first element of a policy statement is parsed here image

This we have to work on!

File: libraries/aws_s3_bucket_policy.rb (hunk 1 above)
Comment (vsuzdaltsev) [reply libraries/aws_s3_bucket_policy.rb:36]: but  for now the description  of POLICY returns BUCKET name

Comment (soumyo13) [reply libraries/aws_s3_bucket_policy.rb:36]: Then we have to change the value of the @display_name.

Comment (soumyo13) [inline libraries/aws_s3_bucket_policy.rb:36]:
```suggestion
    "S3 Bucket Policy of #{@display_name}"
```

Comment (soumyo13) [reply libraries/aws_s3_bucket_policy.rb:36]: We can do like this.

**Review by soumyo13 — APPROVED** [review]

**Pull request merged**